src/python/ripe_demo/static/js/main.js

```diff
@@ -4,7 +4,7 @@
  */
 var FACES = ["side", "top", "front"];
 
-window.onload = function() {

```

Comment:
I think you need to `npm i`, you aren't using the latest dependencies that's why this happens. In any case, if something like this must be done in the future, please include it in a separate PR.